contribute/process/pr-workflow-developer-doc.md

```diff
@@ -0,0 +1,40 @@
+# Contributing a Patch to Developer Documentation
+
+Developer documentation is available under the openebs/openebs/contribute folder. 
+
+The developer documenation includes:
+- Architecture and Design documentation
+- Technical or Research Notes
+- FAQ
+- Process Documentation 
+- Generic / Miscellaneous Notes 
+
+Anything that will help other developers!
+
+## Submit an Issue describing your proposed change.
+
+Some general guidelines when submitting issues for developer documentation:
+- If the proposed change requires an update to the existing page, please provide a link to the page in the issue. 
+
+You can also help with some existing issues under this category available at [developer documentation issues list](https://github.com/openebs/openebs/labels/documentation%2Fdevel)
+
+## Create your development branch. 
+
+- Fork the openebs repository and rebase with master
```

Comment:
rebase? not familiar with the term...is it merge?

Reply:
modified the sentence a bit, however rebase should be also ok..